src/Aspire.Hosting.Oracle/OracleDatabaseBuilderExtensions.cs

```diff
@@ -58,18 +58,20 @@ public static IResourceBuilder<OracleDatabaseResource> AddDatabase(this IResourc
     /// </summary>
     /// <param name="builder">The resource builder.</param>
     /// <param name="name">The name of the volume. Defaults to an auto-generated name based on the application and resource names.</param>
+    /// <param name="isReadOnly">A flag that indicates if this is a read-only volume.</param>
     /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
-    public static IResourceBuilder<OracleDatabaseServerResource> WithDataVolume(this IResourceBuilder<OracleDatabaseServerResource> builder, string? name = null)
-        => builder.WithVolume(name ?? VolumeNameGenerator.CreateVolumeName(builder, "data"), "/opt/oracle/oradata", true);
+    public static IResourceBuilder<OracleDatabaseServerResource> WithDataVolume(this IResourceBuilder<OracleDatabaseServerResource> builder, string? name = null, bool isReadOnly = false)
+        => builder.WithVolume(name ?? VolumeNameGenerator.CreateVolumeName(builder, "data"), "/opt/oracle/oradata", isReadOnly);
```

Comment:
This will be a source breaking change in 8.1, right? Default parameters are fine to add, but it changes the default. If this change is important, perhaps we need it in 8.0
@eerhardt

Reply:
From my perspective the current default is hard broken - the default Oracle image does not come up if the data volume is mounted as read-only. I'll add a bit more detail on the linked issue.

Reply:
If the current default is "hard broken", why are we making it an option at all? If someone passes `isReadOnly: true` wouldn't they be just as broken?

Reply:
@cmdkeen I have the same question..?

Reply:
If it isn't possible for the mount to be read only we should remove the read only option entirely.

@DamianEdwards do you know if the other container types we have any issues with read only volumes?

Reply:
I believe some of them work if the volume already exists and has been initialized by a container of the same image already. The scenario is multiple containers using the same volume but only one has write access. That was the only reason we had an option for read only data volumes.

Reply:
OK thanks. Let me just confirm that also works with Oracle here and if it does we should keep the parameter (with `isReadOnly = false` as the default).